### ISSUE TITLE
Lagrange not starting

### ISSUE THREAD

**Issue by malpaw** [flathub/fi.skyjake.Lagrange#5]
Two days ago, after updating flatpak libraries, Lagrange started to break during startup:
```
$ flatpak run fi.skyjake.Lagrange 
ldconfig: Can't create temporary cache file /run/ld-so-cache-dir/115671a1473fffe5a80d9cfee072060d6d84149c277760b987719ab85ab75062.6AGYm1~: Permission denied
error: ldconfig failed, exit status 256
$
```
With this update I started also getting warnings about 20.08 runtime end-of-life (#3).
However, after #3 had been closed and my libraries updated to 22.08, the ldconfig error did not stop appearing.
I have tried re-installing almost everything with no success.

**Comment by skyjake** [comment]
The error message points to a problem with the flatpak tool itself. After searching for related discussions, nobody seemed to have a known fix apart from maybe upgrading flatpak itself to a newer version.

**Comment by malpaw** [comment]
I did additional research and didn't find anything helpful.
But, I managed to check the upgrade on another machine with the same distro and from the same version and this time it worked without any issues. I also performed clean installation on a VM - also without issues. 
Thus a problem with my flatpak installation is the most likely explanation, even though re-installing flatpak does not help.

Thanks for checking! I am closing this issue.
